app/helpers/devise_helper.rb

```diff
@@ -0,0 +1,34 @@
+module DeviseHelper
+  def devise_error_messages!
+    return '' if resource.errors.empty?
+    html = html_var
+    html.html_safe
```

Comment:
why not make inline? e.g.: `html_var.html_safe`

Reply:
sorry, didn`t  noticed that, will improve.